zanshincli/main.py

```diff
@@ -787,18 +789,16 @@ def summary_alert(organization_id: UUID = typer.Argument(..., help="UUID of the
     Gets a summary of the current state of alerts for an organization, both in total and broken down by scan target.
     """
     client = Client(profile=global_options['profile'])
-
-    output_iterable(client.get_alert_summaries(organization_id, scan_target_id), empty=0)
-
+    dump_json(client.get_alert_summaries(organization_id, scan_target_id))
+ 
 @summary_app.command(name='alert_following')
 def summary_alert_following(organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
                             following_ids: Optional[List[UUID]] = typer.Option(None, help="Only summarize alerts from the specified following, defaults to all.")):
     """
     Gets a summary of the current state of alerts for followed organizations.
     """
     client = Client(profile=global_options['profile'])
-
-    output_iterable(client.get_following_alert_summaries(organization_id, following_ids), empty=0)
+    dump_json(client.get_following_alert_summaries(organization_id, following_ids), empty=0)
```

Comment:
Acho que o `empty=0` não existe como parametro no `dump_json`, esta certo?

Reply:
Ah, good catch! My bad. I made a mistake just prior to sending this out. I've now run pylint and fixed this and other errors. Thanks!